src/relationUtils.ts

```diff
@@ -276,3 +280,118 @@ export function toJson(output: Relation[] | ArrowRelation[]): any {
 
   return result;
 }
+
+export function filterRelations(relations: ArrowRelation[], keys: string[]) {
```

Comment:
does this only work where `keys[]` is the prefix? If so, might be worth calling out in a comment/docstring and/or renaming

Reply:
Not sure that I understand what you mean. It filters relations bases on the passed keys.

If you have relations like
`/:foo/:baz/Int64/:bar`
`/:foo/:baz/String/:boo`
`/:foo/String/:boo`

and filter like `filterRelations(rels, [":foo", ":baz"])` you get only the first two

Reply:
That makes sense.  I meant the keys had to be at the start of the relation - so in this case you couldn't filter on just `:baz` and get the same two relations.

Reply:
Yeah, that's a bit confusing. I'm trying to move utils/helpers that often used. This's one for example is handy when you get relations like `/:output/:mystuff/:here` and you just want only those that are starting from `/:output` and you want to get only `/:mystuf/:here` part of it.

Or you have `/:plot/:vega/:blah/:blah1` and you just want the blah part.

We'll refactor/standardize/document these things later on.